README.md

```diff
@@ -59,198 +79,217 @@ You can use these environment variables in your workflow as follows:
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 
 ### Storing the connection details using single Config Token
-1. Make sure JFrog CLI is installed on your local machine by running ```jf -v```.
-2. Configure the details of the JFrog platform by running ```jf c add```.
-3. Export the details of the JFrog platform you configured, using the server ID you chose. Do this by running ```jf c export <SERVER ID>```.
+
+1. Make sure JFrog CLI is installed on your local machine by running `jf -v`.
+2. Configure the details of the JFrog platform by running `jf c add`.
+3. Export the details of the JFrog platform you configured, using the server ID you chose. Do this by running `jf c export <SERVER ID>`.
 4. Copy the generated Config Token to the clipboard and save it as a secret on GitHub.
 
 To use the saved JFrog platform configuration in the workflow, all you need to do it to expose the secret to the workflow.
-The secret should be exposed as an environment variable with the *JF_ENV_* prefix.
+The secret should be exposed as an environment variable with the _JF*ENV*_ prefix.
 Here's how you do this:
+
 ```yml
 - uses: jfrog/setup-jfrog-cli@v3
   env:
-    JF_ENV_1: ${{ secrets.JF_SECRET_ENV_1 }}
+      JF_ENV_1: ${{ secrets.JF_SECRET_ENV_1 }}
 - run: |
-    # Ping the server
-    jf rt ping
+      # Ping the server
+      jf rt ping
 ```
-As you can see in the example above, we created a secret named *JF_SECRET_ENV_1* and exposed it to the workflow 
-as the *JF_ENV_1* environment variable. That's it - the ping command will now ping the configured Artifactory server.
+
+As you can see in the example above, we created a secret named _JF_SECRET_ENV_1_ and exposed it to the workflow
+as the _JF_ENV_1_ environment variable. That's it - the ping command will now ping the configured Artifactory server.
 
 If you have multiple Config Tokens as secrets, you can use all of them in the workflow as follows:
+
 ```yml
 - uses: jfrog/setup-jfrog-cli@v3
   env:
-    JF_ENV_1: ${{ secrets.JF_SECRET_ENV_1 }}
-    JF_ENV_2: ${{ secrets.JF_SECRET_ENV_2 }}
+      JF_ENV_1: ${{ secrets.JF_SECRET_ENV_1 }}
+      JF_ENV_2: ${{ secrets.JF_SECRET_ENV_2 }}
 - run: |
-    # Set the JFrog configuration to use by providing the server ID (configured by the 'jf c add' command).
-    jf c use local-1
-    # Ping local-1 Artifactory server
-    jf rt ping
-    # Now use the second sever configuration exposed to the Action.
-    jf c use local-2
-    # Ping local-2 Artifactory server
-    jf rt ping
+      # Set the JFrog configuration to use by providing the server ID (configured by the 'jf c add' command).
```

Comment:
Maybe rephrase for better understanding:
Set the utilized JFrog configuration by providing the server ID (configured by the 'jf c add' command).